atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -97,7 +97,11 @@ public void continueOrPropagate(RetryableException ex) {
 
         checkAndHandleFailure(ex);
         if (retryBehaviour.shouldBackoffAndTryOtherNodes()) {
-            pauseForBackOff(ex);
+            if (failoverCount.get() % servers.size() == 0) {
+                pauseForBackoff(ex, 1000L);
```

Comment:
We should make this backoff configurable. 1 second might be too conservative for a default. We could figure out how long, on average, a leadership election takes, and then wait for that amount of time.